staging/src/k8s.io/kubectl/pkg/cmd/cmd.go

```diff
@@ -165,6 +165,25 @@ __kubectl_get_resource()
     __kubectl_parse_get "${nouns[${#nouns[@]} -1]}"
 }
 
```

Comment:
Do you mind to describe your function a little bit?  It seems now we have __kubectl_get_resources() and __kubectl_get_resource() :)

Reply:
added some comments, please have a look.